audio_offset_finder/audio_offset_finder.py

```diff
@@ -57,7 +57,7 @@ def find_offset(file1, file2, fs=8000, trim=60*15, hop_length=128, win_length=25
     score = (c[max_k_index] - np.mean(c)) / np.std(c) # standard score of peak
     os.remove(tmp1)
     os.remove(tmp2)
-    return offset, score
+    return {"offset":offset, "score":score, "correlation":c}
```

Comment:
This reminds me of this issue https://github.com/bbc/audio-offset-finder/pull/9/files which suggested adding a JSON format output. The change here would make that simple to implement. Perhaps for a separate PR though?

Reply:
Definitely a distinct issue.